src/main/webapp/app/shared/metis/posting-header/post-header/post-header.component.html

```diff
@@ -1,25 +1,39 @@
-<div class="row justify-content-between my-1">
-    <div class="col-auto pe-0">
-        <span class="posting-header header-author-date">
+<div class="d-flex justify-content-between my-1">
+    <div class="align-items-center">
+        <span class="fs-medium header-author-date">
```

Comment:
See Feedback in src/main/webapp/app/shared/metis/posting-header/answer-post-header/answer-post-header.component.html